configs/RRIOT_F722/config.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU STM32F7X2
+
+#define BOARD_NAME RRIOT_F722
+#define MANUFACTURER_ID EASP
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP388
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+#define MOTOR1_PIN PA9
+#define MOTOR2_PIN PA8
+#define MOTOR3_PIN PC9
+#define MOTOR4_PIN PC8
+#define MOTOR5_PIN PB5
+#define MOTOR6_PIN PB4
+
+#define UART1_TX_PIN PB6
+#define UART1_RX_PIN PB7
+#define UART2_TX_PIN PA2
+#define UART2_RX_PIN PA3
+#define UART3_TX_PIN PB10
+#define UART3_RX_PIN PB11
+#define UART4_TX_PIN PA0
+#define UART4_RX_PIN PA1
+#define UART5_TX_PIN PC12
+#define UART5_RX_PIN PD2
+#define UART6_TX_PIN PC6
+#define UART6_RX_PIN PC7
+
+#define I2C1_SCL_PIN PB8
+#define I2C1_SDA_PIN PB9
+
+#define SPI1_SCK_PIN PA5
+#define SPI1_SDI_PIN PA6
+#define SPI1_SDO_PIN PA7
+#define SPI2_SCK_PIN PB13
+#define SPI2_SDI_PIN PB14
+#define SPI2_SDO_PIN PB15
+#define SPI3_SCK_PIN PC10
+#define SPI3_SDI_PIN PC11
+#define SPI3_SDO_PIN PB2
+
+#define CAMERA_CONTROL_PIN PA10
+#define ADC_VBAT_PIN PC0
+#define ADC_CURR_PIN PC1
+#define FLASH_CS_PIN PA15
+#define MAX7456_SPI_CS_PIN PB12
+#define GYRO_1_EXTI_PIN PB1
+#define GYRO_1_CS_PIN PA4
+#define LED0_PIN PC14
+#define BEEPER_PIN PC13
+
+#define TIMER_PIN_MAPPING       \
+    TIMER_PIN_MAP(0, PA9, 1, 0) \
+    TIMER_PIN_MAP(1, PA8, 1, 0) \
+    TIMER_PIN_MAP(2, PC9, 1, 0) \
+    TIMER_PIN_MAP(3, PC8, 1, 0) \
+    TIMER_PIN_MAP(4, PB5, 1, 0) \
+    TIMER_PIN_MAP(5, PB4, 1, 0) \
+    TIMER_PIN_MAP(6, PA10, 1, -1)
+
+#define ADC1_DMA_OPT 0
+
+#define GYRO_1_SPI_INSTANCE SPI1
+#define MAX7456_SPI_INSTANCE SPI2
+#define FLASH_SPI_INSTANCE SPI3
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
```

Comment:
You can consider using these calibration values to get voltage and current meter set up and detecting correct values. (In case they differ from the default value)
```c
#define DEFAULT_CURRENT_METER_SCALE 400
#define DEFAULT_CURRENT_METER_OFFSET 0

#define DEFAULT_VOLTAGE_METER_SCALE 110
#define DEFAULT_VOLTAGE_METER_DIVIDER 10
#define DEFAULT_VOLTAGE_METER_MULTIPLIER 1
```

Reply:
Adding these in, thanks for the suggestion!